src/Diff/DiffOpFactory.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Wikibase;
+
+use Diff\DiffOp\DiffOp;
+use Diff\DiffOpFactory as DiffDiffOpFactory;
```

Comment:
This was the minor alteration ;) So that this class doesn't have to be called WikibaseDiffOpFactory